### ISSUE TITLE
[LibraryAddition] Indicating clearly that the precursor mass is optional.

### ISSUE THREAD

**Issue by lfnothias** [CCMS-UCSD/GNPS_Workflows#268]
In some cases, users are providing inaccurate mass for the precursor ion. So it is better to indicate that it is optional, so the worklflow extracts that mz value from the actual file.
Like "precursor mass (optional)"

**Comment by robinschmid** [comment]
Can you always extract it from raw data?

**Comment by lfnothias** [comment]
Yes, this is what is happening when the user doesn't specify it. So this  is why I think it should be the default so we limit inaccurate mass specified by the user (which happens quite often).

**Comment by mwang87** [comment]
Will update library addition input for to indicate this.
